### DISCUSSION TITLE
Only updating every 45 - 3 hours

### DISCUSSION THREAD

**Discussion by DiabeticDonkey100** [NightscoutFoundation/xDrip#3822] [Bug reports]
Im using stelo
![Screenshot_20241218_103001_xDrip+](https://github.com/user-attachments/assets/bd6214c3-c984-4108-8307-b1f115ae7937)
![Screenshot_20241218_103010_xDrip+](https://github.com/user-attachments/assets/0f64cd4e-0c62-4d92-b375-e0a0432ba8b0)

**Comment by Navid200** [comment]
These are the instructions:  https://navid200.github.io/xDrip/docs/Dexcom/G7.html

As stated in there, if you want to use xDrip as a collector, you will have to uninstall Dexcom app.

**Reply by DiabeticDonkey100** [reply]
I did uninstall the dexcom app but it didn't help

**Reply by Navid200** [reply]
xDrip is not a turn key solution.
If you expect to flip a switch and everything will work, you will be disappointed.

After uninstalling Dex app, you will have to wait 15 minutes for it to start looking for connectivity.

You should forget all devices starting with DX01 and DX02.
And you need to move any used sensors more than 10 meters away.

Now, go to the status page and wait for a pair request.
You need to stay there as long as it shows Hunting Transmitter and you need to make sure that xDrip is scanning.
